programs/oxipng.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  config,
+  mkFormatterModule,
+  ...
+}:
+let
+  cfg = config.programs.oxipng;
+in
+{
+  meta.maintainers = [ "nim65s" ];
+
+  imports = [
+    (mkFormatterModule {
+      name = "oxipng";
+      includes = [ "*.png" ];
+    })
+  ];
+
+  options.programs.oxipng = {
+    alpha = lib.mkEnableOption "Perform additional optimization on images with an alpha channel";
+    opt = lib.mkOption {
+      description = "Set the optimization level preset";
+      type = with lib.types; nullOr str;
```

```suggestion
      type = with lib.types; str;
```
Since it's not using lib.optionals below. Or use optionals below.

---

Yes, there is no need for options here, thanks !